fineract-provider/src/test/java/org/apache/fineract/infrastructure/event/external/service/ExternalEventServiceTest.java

```diff
@@ -122,7 +131,8 @@ public void testPostEventShouldWorkWithRegularEvent() throws IOException {
         given(idempotencyKeyGenerator.generate(event)).willReturn(idempotencyKey);
         given(serializerFactory.create(event)).willReturn(eventSerializer);
         given(eventSerializer.getSupportedSchema()).will(invocation -> LoanAccountDataV1.class);
-        given(eventSerializer.serialize(event)).willReturn(data);
+        given(eventSerializer.toAvroDTO(event)).willReturn(new LoanAccountDataV1());
+        given(byteBufferConverter.convert(any(ByteBuffer.class))).willReturn(data);
```

Comment:
I might be wrong, but i dont see any places where it was verified the data enricher was called and provided the right value...